features/ei-solution-features/org.wso2.carbon.ei.solutions.ei.analytics.feature/pom.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>ei-solution-features</artifactId>
+        <groupId>org.wso2.analytics.solutions</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <name>WSO2 Stream Processor - EI Analytics Solution Feature</name>
+    <artifactId>org.wso2.carbon.ei.solutions.ei.analytics.feature</artifactId>
+
+
+    <dependencies>
+        <!--Widgets-->
+        <dependency>
+            <groupId>org.wso2.analytics.solutions</groupId>
+            <artifactId>org.wso2.carbon.analytics.solutions.ei.analytics.horizontal.bar.chart</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
+            <type>zip</type>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.analytics.solutions</groupId>
+            <artifactId>org.wso2.carbon.analytics.solutions.ei.analytics.mediator.properties</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```

Comment:
Please remove version tag. Version should be coming from the project pom file. Kindly fix in other places as well (in this pom file).

Reply:
Fixed in d0663ee31e3530ba950220f24e7c317eb333d7c8